test/Core.Test/OrganizationFeatures/Groups/CreateGroupCommandTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Data;
+using Bit.Core.OrganizationFeatures.Groups;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Test.AutoFixture.OrganizationFixtures;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.OrganizationFeatures.Groups;
+
+[SutProviderCustomize]
+public class CreateGroupCommandTests
+{
+    [Theory, OrganizationCustomize(UseGroups = true), BitAutoData]
+    public async Task CreateGroup_Success(SutProvider<CreateGroupCommand> sutProvider, Organization organization, Group group)
+    {
+        sutProvider.GetDependency<IOrganizationRepository>().GetByIdAsync(group.OrganizationId).Returns(organization);
+        var utcNow = DateTime.UtcNow;
+
+        await sutProvider.Sut.CreateGroupAsync(group);
+
+        await sutProvider.GetDependency<IGroupRepository>().Received(1).CreateAsync(group);
+        await sutProvider.GetDependency<IEventService>().Received(1).LogGroupEventAsync(group, Enums.EventType.Group_Created);
```

Comment:
I think you should also assert the call to `ReferenceEventService` in each happy path test.